packages/coreutils/src/time.ts

```diff
@@ -17,9 +17,7 @@ const UNITS: { name: Intl.RelativeTimeFormatUnit; milliseconds: number }[] = [
  * The namespace for date functions.
  */
 export namespace Time {
-  // Intl.RelativeTimeFormatStyle contains these, but it requires `ES2020.Intl`.
-  // We currently compile to an `ES2018` target.
-  export type HumanStyle = 'long' | 'short' | 'narrow';
+  export type HumanStyle = Intl.ResolvedRelativeTimeFormatOptions["style"];
```

Comment:
This could target `Intl.RelativeTimeFormatStyle` as originally proposed. The only downside to this is that there is a proposal to rename the RelativeTimeFormat option types to bring them in line with the Intl namespace convention (which would be `Intl.RelativeTimeFormatOptionsStyle`), so this type name may be removed or deprecated in a future TypeScript release.

Reply:
It looks like it does not compile:

```
  ../coreutils/lib/time.d.ts(5,28): error TS2724: 'Intl' has no exported member named 'ResolvedRelativeTimeFormatOptions'. Did you mean 'ResolvedDateTimeFormatOptions'?
```

Reply:
What was missing from the PR (mea culpa) was the default library set in tsconfigbase. This seems like an acceptable approach, given that Jupyter explicitly relies on features defined in those libraries.